tests/unit/AccountService.js

```diff
@@ -0,0 +1,23 @@
+describe('AccountService tests', function(){
+  var accountService = new AccountService();
+
+  describe('--> SignUp tests', function(){
+    it('--> SignUp false', function(){
```

Comment:
It's VERY important that the tests read as clear as possible and make sense even for someone who never looked at your code. Imagine you have on your project thousands of tests written by different people across the years and you need to update some tests that say "SignUp false" etc. I bet  it will not be easy or pleasent.

Write the tests something like:

"Should <description of what should happen" when <describe the condition in which the thing should happen>".
Eg "Signup should return false when an invalid username is passed in" etc